storage/fuzz/fuzz_targets/adb_current_operations.rs

```diff
@@ -5,7 +5,7 @@ use commonware_cryptography::Sha256;
 use commonware_runtime::{buffer::PoolRef, deterministic, Runner};
 use commonware_storage::{
     adb::current::{Config, Current},
-    mmr::hasher::{Hasher, Standard},
+    mmr::{Hasher as MmrHasher, StandardHasher as Standard},
```

Comment:
nit: feels like `MmrHasher`/`StandardHasher` is a better combo than `MmrHasher`/`Standard`